R/arc-read.R

```diff
@@ -2,27 +2,28 @@
 #'
 #' [arc_read()] combines the functionality of [arc_open()] with [arc_select()]
 #' or [arc_raster()] to read an ArcGIS FeatureLayer, Table, or ImageServer to a
-#' `sf` object or object of class `SpatRaster`. `n_max` defaults to 10000 to
-#' avoid unintentionally reading an entire layer with a very large number of
-#' features.
+#' `sf` object or object of class `SpatRaster`. Optionally set, check, or modify
+#' names for the returned data frame or sf object using the `col_names` and
+#' `name_repair` parameters.
 #'
 #' @inheritParams arc_open
-#' @param col_select Columns to select. Alternate argument for specifying fields
-#'   if fields is `NULL`.
-#' @param col_names If `TRUE`, use the default column names for the feature. If
-#'   `col_names` is a character function with the same length as the number of
-#'   columns in the layer, the default names are replaced with the character
-#'   vector. If `col_names` is one less than the length of the default or if
-#'   `col_names` if `FALSE`, the existing sf column name is retained. If
-#'   `col_names` is the string "alias", names are set to match the available
+#' @param col_names Default `NULL`. If `TRUE`, use the default column names for
+#'   the feature. If `col_names` is a character vector with the same length as
+#'   the number of columns in the layer, the default names are replaced with the
+#'   new names. If the length of `col_names` is one less than the length of the
+#'   default column names, the existing sf column name is retained. If
+#'   `col_names` is the string `"alias"`, names are set to match the available
```

Comment:
Can we change `one less` to `one fewer`?

Reply:
I went ahead and changed it with revisions to indicate it was fewer names rather than fewer length. Seems like distance is an area of common exception from the standard grammar of fewer vs. less: https://www.merriam-webster.com/grammar/fewer-vs-less

Reply:
I generally stick with the general rule of thumb if its discrete (e.g. you can't have half of a name!) use fewer otherwise, less. Thanks for appeasing me lol